### ISSUE TITLE
pdf version - missing images

### ISSUE THREAD

**Issue by emmapa** [Academany/FabAcademy-Handbook#3]
there are not images in the pdf version.

**Comment by TheBeachLab** [comment]
Lets kill the PDF version

**Comment by emmapa** [comment]
I wouldn't do it, it's the only printable version. yes, sometimes I still print on paper.

**Comment by TheBeachLab** [comment]
Exactly, because people prints it and it is a live document. If you print it it is obsolete.

**Comment by emmapa** [comment]
it also means that you can have access to it only if you are connected. I had to print the document today to review it on the train. I like pdf. but I understand your point.
